sdk/src/main/java/io/radar/sdk/RadarLocationManager.kt

```diff
@@ -403,6 +403,8 @@ internal class RadarLocationManager(
                         "locationPowerSaveMode = ${batteryState.getPowerLocationPowerSaveModeString()}; " +
                         "isDozeMode = ${batteryState.isDeviceIdleMode}"
             )
+        } else {
+            logger.d("Handling location | location = $location")
         }
```

Comment:
nit: should we have a space below this `else` block to denote the below `if` as a separate conditional statement?

Reply:
I generally avoid extra spacing - but that seems to be a pattern used throughout this class, so I can add it. This will be something we want to document in a style/contributing guide.